src/index.html

```diff
@@ -22,6 +22,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="https://apple.com"
```

Comment:
The 'is-active' class should be 'nav__link--active' to follow the BEM methodology as per the checklist. The modifier should be separated from the block or element name by two hyphens.